src/scss/_responsive-flat.scss

```diff
@@ -0,0 +1,64 @@
+////
+/// @group o-table
+/// @link http://registry.origami.ft.com/components/o-table
+////
+
+/// Make a table 'responsive-flat'
+@mixin oTableResponsive-flat {
+	*,
```

Comment:
Unsure if using `*` is the best way to do this... This would generate a `.o-table--responsive-flat *` selector? @AlbertoElias do you have any thoughts?